### PR TITLE
initial debianization

### PR DESCRIPTION
**Pull request by noxiouz**
- [x] Upstart/Systemd
- [x] Default configs?

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	dh $@ --with systemd
+
+override_dh_auto_build:
+	make -j build/insomnia
+
+override_dh_auto_test:
+	echo "skip tests"
+
+override_dh_auto_install:
+	install -d $(DEB_DESTDIR)etc/sonm/
```
Comment (antmat) [inline debian/rules]: Looks like you have only configs in your package

File: debian/rules [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+
+%:
+	dh $@ --with systemd
+
+override_dh_auto_build:
+	make -j build/insomnia
+
+override_dh_auto_test:
+	echo "skip tests"
```
Comment (antmat) [inline debian/rules]: why?

File: debian/rules [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+override_dh_auto_install:
+	install -d $(DEB_DESTDIR)etc/sonm/
+	install -m644 miner.yaml $(DEB_DESTDIR)etc/sonm/miner-default.yaml
+	install -m644 hub.yaml $(DEB_DESTDIR)etc/sonm/hub-default.yaml
+
+override_dh_systemd_start:
+	echo "Not running dh_systemd_start"
+
+override_dh_auto_clean:
+	rm -rf etc/
```
Comment (antmat) [inline debian/rules]: Are you sure?

Comment (noxiouz) [reply debian/rules]: Yep) The trick is not to forget / =)

File: debian/rules (hunk 2 above)
Comment (noxiouz) [reply debian/rules]: requires lots of stuff to do. For example, generate mocks, which is tricky under debuild :)
At the same time we run tests in Travis before debuild

**Pull request merged**